### PR TITLE
activating signals for property changes improved

### PR DESCRIPTION
**Pull request by devmil**
no explicit property name passing needed any more

### CODE REVIEW

File: src/net/Qml.Net/Signals.cs [hunk 1]
```diff
@@ -34,6 +35,13 @@ public static bool ActivateSignal(this object instance, string signalName, param
             return true;
         }
 
+        public static bool ActivatePropertyChangedSignal(this object instance, [CallerMemberName] string propertyName = "")
+        {
+            var signalName = $"{propertyName}Changed";
```
Comment (pauldotknopf) [inline src/net/Qml.Net/Signals.cs]: There is no fixed-convention on what this signal name would be called.

File: src/net/Qml.Net.Tests/Qml/SignalTests.cs [hunk 2]
```diff
@@ -31,6 +31,20 @@ public virtual void TestMethodWithArgs(string arg1, int arg2)
             {
                 
             }
+
+            private string _somePropertyValue = "";
+
+            [NotifySignal]
```
Comment (pauldotknopf) [inline src/net/Qml.Net.Tests/Qml/SignalTests.cs]: This attribute takes a "signal name" property that could have any value. In later code, you assume the value.

**Comment by pauldotknopf** [comment]
I don't know if this is a good way to do this.

I am assuming you'd like a consistent/safe way to invoke property changes from the MVVM stuff? In that case, I think you should validate/ensure the property names are as you expect, up front, when types are registered. Then, from MVVM, you make assumptions.

**Comment by devmil** [comment]
Yeah, right. When the signal name is manually set this extension doesn't make any sense

**Pull request closed without merging**